builder/bundleset.go

```diff
@@ -297,7 +301,11 @@ func parseBundle(contents []byte) (*bundle, error) {
 	}
 
 	b.DirectIncludes = includes
-	b.DirectPackages = packages
+	b.Files = make(map[string]bool)
```

Comment:
It looks like this also gets init'd at L196 of `chroots.go` (aka, first line of `resolveFilesForBundle`). What do you think about leaving it nil here (i.e., don't use memory until we need it)? Or not re-init'ing it in `resolveFilesForBundle`.  I suggested leaving it nil first, though, just 'cause none of the `mixer bundle` commands use this field. Guess it's a pretty small amount of memory, though :)

Reply:
Fair. I think I was debugging the panic I got from not initing it at all and I accidentally put it in two places.

Reply:
done.